nexus/tests/integration_tests/saml.rs

```diff
@@ -32,7 +32,7 @@ async fn test_create_a_saml_idp(cptestctx: &ControlPlaneTestContext) {
     let client = &cptestctx.external_client;
 
     const SILO_NAME: &str = "saml-silo";
-    create_silo(&client, SILO_NAME, true, shared::UserProvisionType::Fixed)
+    create_silo(&client, SILO_NAME, true, shared::SiloIdentityMode::LocalOnly)
```

Comment:
this shouldn't be LocalOnly, right? one of the future planned changes is to reject configuring an identity provider if it doesn't match the identity mode, which means this test will be broken.

Reply:
Yeah, good call.  I was separately trying to sneak in one more change that validates that and updates several of these tests accordingly.  That's coming shortly.

Reply:
Fixed in 394c3be9.